tools/send-to-caas/bulkpublisher.beta.html

```diff
@@ -132,7 +132,7 @@ <h4>Advanced Options</h4>
                 </div>
                 <div id="use-preview-cb" class="field checkbox">
                   <input type="checkbox" id="usePreview" name="usePreview" />
-                  <label for="usePreview">Use Preview Content (*.page)<span class="help use-preview" title="Help about this functionality">(?)</span></label>
+                  <label for="usePreview">Use Preview Content (*.stage.*)<span class="help use-preview" title="Help about this functionality">(?)</span></label>
```

Comment:
This is good as it clarifies that the preview environment is a typical stage domain.